### ISSUE TITLE
a question about the loss function LF

### ISSUE THREAD

**Issue by Fairydetail** [sniklaus/pytorch-sepconv#52]
Hi, I used the reimplemntation of the sepconv-slomo, but I noticed that the loss might be wrong. I guess it should be 
**loss = torch.nn.MSELoss()(outputFeatures, targetFeatures)**. Is it right?
(from https://github.com/martkartasev/sepconv/blob/master/src/loss.py):

class VggLoss(nn.Module):

    def __init__(self, device=torch.device('cpu')):
        super(VggLoss, self).__init__()

        model = torchvision.models.vgg19(pretrained=True)

        self.features = nn.Sequential(
            # stop at relu4_4 (-10)
            *list(model.features.children())[:-10]
        )

        for param in self.features.parameters():
            param.requires_grad = False

    def forward(self, output, target):

        output_features = self.features(output)
        target_features = self.features(target)

        loss = torch.norm(output_features - target_features, 2)

        return loss

**Comment by sniklaus** [comment]
You are correct that `torch.nn.MSELoss` would be more true to our paper. The two are related though as shown below.

```
>>> a = torch.rand(10, 3, 10, 10)
>>> b = torch.rand(10, 3, 10, 10)
>>> torch.norm(a-b,2)**2
tensor(486.0934)
>>> torch.nn.MSELoss(reduction='sum')(a, b)
tensor(486.0934)
```

So one could probably use `torch.norm(output_features - target_features, 2)**2`, but I am not sure how the `**2` affects the training (your mileage may vary, it could be better or it could be worse).

On a different note since you are working on reimplementing SepConv, you might find this useful: http://sniklaus.com/resepconv

**Comment by Fairydetail** [comment]
Thanks a lot for your reply.
